v21.2/change-data-capture-overview.md

```diff
@@ -25,6 +25,26 @@ The main feature of CDC is the changefeed, which targets an allowlist of tables,
 
 See [Ordering Guarantees](use-changefeeds.html#ordering-guarantees) for detail on CockroachDB's at-least-once-delivery-guarantee as well as explanation on how rows are emitted.
 
+## How does an Enterprise changefeed work?
+
+When an {{ site.data.products.enterprise }} changefeed is started on a node, that node becomes the _coordinator_ for the changefeed job (**Node 2** in the diagram). The coordinator node acts as an administrator: keeping track of all other nodes during job execution and the changefeed work as it completes. In the unlikely event that the changefeed’s coordinating node were to fail during the job, that role will move to a different node and the changefeed will restart from the last checkpoint.
+
+The changefeed job will run across nodes in the cluster to access changed data in the watched table. Typically, the [leaseholder](architecture/replication-layer.html#leases) for a particular range (or the range’s replica) determines which node emits the changefeed data.
+
+Each node uses its aggregator processors to send back checkpoint progress to the coordinator that gathers this information to update the high-water mark timestamp. The high-water mark acts as a checkpoint for the changefeed’s job progress, and guarantees that all changes before (or at) the timestamp have been emitted. If restarted, the changefeed will send duplicate messages from the high-water mark timestamp. For more detail on when messages are emitted, see [Ordering Guarantees](use-changefeeds.html#ordering-guarantees).
```

Comment:
"If restarted, the changefeed will send duplicate messages from the high-water mark timestamp."

i think this sentence could be worded a bit more clearly, maybe "If restarted, the changefeed will send duplicate messages starting at the high watermark time to the current time"